HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -744,4 +744,21 @@ Patch "Infinite Boost Gauge" by "Hyper"
         
         0xE9, 0xC2, 0x01, 0x00, 0x00
     );
+}
+
+Patch "Raise FPS Cap to 300 (experimental)" by "NM"
+{
+    WriteAsmHook(
+        // 0x43960000 == 300.0f
+        @"
+            mov dword ptr [rcx+0x14],0x43960000
+            mov rbx,rcx
+            movss dword ptr [rcx+144h],xmm1
+        ",
```

Comment:
```suggestion
    /* 0x43960000 == 300.0f */
    WriteAsmHook(
        @"
            mov dword ptr [rcx+0x14], 0x43960000
            mov rbx, rcx
            movss dword ptr [rcx+144h], xmm1
        ",
```

Reply:
That comment could be moved to an ASM comment instead, if you want to be explicit about what the value is;
```
mov dword ptr [rcx+0x14], 0x43960000 ; 300.0f
```